blobstore/src/main/java/org/jclouds/blobstore/config/LocalBlobStore.java

```diff
@@ -364,7 +364,9 @@ private SortedSet<StorageMetadata> extractCommonPrefixes(SortedSet<StorageMetada
             o = prefix + o;
          }
          md.setName(o + delimiter);
-         contents.add(md);
+         if (!contents.contains(md)) {
+            contents.add(md);
+         }
```

Comment:
`contents` is a `Set` so this is the same as before.  Could you revert it?

Reply:
It is reverted now. I thought that item will be replaced if I `add` an existing item in `Set`, but actually it will be ignored.